lib/utils/disk.go

```diff
@@ -53,10 +53,7 @@ func FreeDiskWithReserve(dir string, reservedFreeDisk uint64) (uint64, error) {
 	if err != nil {
 		return 0, trace.Wrap(err)
 	}
-	if stat.Bsize < 0 {
-		return 0, trace.Errorf("invalid size")
-	}
-	avail := stat.Bavail * uint64(stat.Bsize)
+	avail := uint64(stat.Bavail) * uint64(stat.Bsize)
```

Comment:
This change related to this comment https://github.com/gravitational/teleport/pull/47060#discussion_r1795868236
also I'm locally checking linter for warnings related to`uint64(stat.Bavail)`